config/calico_crds.yml

```diff
@@ -0,0 +1,6 @@
+# The Makefile goals for read-calico-crds-version and read-calico-enterprise-crds-version read from here what branch
```

Comment:
This is probably good as a temporary fix but we will likely want the ability to pin to a release version once we have releases built from the new monorepo.

What do you think @tmjd ?

Reply:
I don't think this is a good fix because it is adding a new file for versions when we already have files for calico and enterprise versions. @rene-dekker why do we need to add a new file for this?
If we need a quick fix I think the only thing that needs to change are the git commands used.

Reply:
I can revert these changes. It will work for master, but if you want to build release-v1.24, the libcalico-go version will point to v3.21.0, while that tag/branch does not exist in projectcalico/calico. So whoever builds that will probably want to tweak it into something like I propose to get un-stuck.

Want me to revert, so we can unblock master?

Reply:
I think in release-v1.24 it would need to be updated to point to the correct branch. Are you saying there is no v3.21.0 branch, I just checked and there is.

Reply:
Yes please revert adding this new file.

Reply:
I kind of misremembered. The branch indeed exists, but it has no monorepo stuff yet: https://github.com/projectcalico/calico/tree/v3.21.0

Reply:
I reverted the changed, other than the git commands for now.